spec/models/user_spec.rb

```diff
@@ -14,4 +14,13 @@
       expect(user).to_not be_valid
     end
   end
+
+  describe 'balance' do
```

Comment:
@marderer `describe '#balance'` from best practices and conventions
See:
`#something` - model's instance method
`.something` - model's relation method